src/installer/BeatPackageCompiler/BeatPackageCompiler.cs

```diff
@@ -215,27 +215,22 @@ static void Main(string[] args)
 
             System.IO.File.WriteAllText(cliShimScriptPath, Resources.GenericCliShim);
 
-            var beatsInstallPath =
-                $"[ProgramFiles{(ap.Is64Bit ? "64" : string.Empty)}Folder]" +
-                Path.Combine(companyName, productSetName);
+            var beatsInstallPath = $"c:\\Program Files\\" +
```

Comment:
Can this still use the program files var so we aren't hard coding the c:\ drive?

Reply:
No... this is what happens, probably because it's used as a property value (and not a property key):
<img width="289" alt="image" src="https://github.com/elastic/elastic-stack-installers/assets/2217774/5b97a742-fa6a-4a67-be21-b146b4407941">